README.md

```diff
@@ -284,9 +284,9 @@ Open `x64 Native Tools Command Prompt for VS2019` and run the following commands
 > md dynawo-project && cd dynawo-project
 > git clone https://github.com/dynawo/dynawo.git
 > cd dynawo
-> cmake -S dynawo/3rdParty -B b-3-p -DCMAKE_INSTALL_PREFIX=../d-3-p -DOPENMODELICA_INSTALL=../OpenModelica/Install -DOPENMODELICA_SRC=../OpenModelica/Source -DOMDEV_HOME=../OMDev -G "Ninja"
+> cmake -S dynawo/3rdParty -B b-3-p -DCMAKE_INSTALL_PREFIX=../d-3-p -DOPENMODELICA_INSTALL=../OpenModelica/Install -DOPENMODELICA_SRC=../OpenModelica/Source -DOMDEV_HOME=../OMDev
```

Comment:
Ninja allows to compile faster why remove it from the example ?

Reply:
ok so we can drop it for now and make it work again later on as it allows to build in parallel. We should open an issue.

Reply:
Done!

https://github.com/dynawo/dynawo/issues/3731
https://github.com/dynawo/dynawo-algorithms/issues/437
https://github.com/dynawo/dynaflow-launcher/issues/879